### PR TITLE
chore(common.groovy): bump helm to latest

### PR DESCRIPTION
**Pull request by vdice**
Requires https://github.com/deis/e2e-runner/pull/102

TODO
 - [x] test in conjunction w/ aforementioned PR

### CODE REVIEW

**Review by mboersma — APPROVED** [review]

File: common.groovy [hunk 1]
```diff
@@ -23,7 +23,7 @@ defaults = [
     webhookURL: 'a53b3a9e-d649-4cff-9997-6c24f07743c8',
   ],
   helm: [
-    version: 'v2.3.1',
+    version: 'v2.5.0',
```
Comment (bacongobbler) [inline common.groovy]: might want to bump to v2.5.1 instead; there were a few pretty serious bugs in 2.5.0.

https://github.com/kubernetes/helm/releases/tag/v2.5.1

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by bacongobbler** [comment]
Sorry for the delay on this review btw!

**Comment by vdice** [comment]
Bumped to latest Helm release `v2.6.0` and tested in conjunction w/ related PR in https://ci.deis.io/job/workflow-chart-e2e/595/console -- I think we should be ready to merge!

**Comment by mboersma** [comment]
@vdice there's [Helm v2.6.1](https://github.com/kubernetes/helm/releases/tag/v2.6.1) now. But updating Helm for Workflow CI isn't a big priority given this week is the [final monthly release](https://deis.com/blog/2017/deis-workflow-final-release/), so we could also just close this--up to you.

**Pull request merged**